SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialWorkerFlags.h

```diff
@@ -20,22 +19,21 @@ class SPATIALGDK_API USpatialWorkerFlags : public UBlueprintFunctionLibrary
 	 * @param OutValue - Value of worker flag
 	 * @return - If worker flag was found.
 	 */
-	UFUNCTION(BlueprintCallable, Category="SpatialOS")
-	static bool GetWorkerFlag(const FString& Name, FString& OutValue);
-	
-	static FOnWorkerFlagsUpdated& GetOnWorkerFlagsUpdated();
+	bool GetWorkerFlag(const FString& Name, FString& OutValue) const;
+
+	FOnWorkerFlagsUpdated& GetOnWorkerFlagsUpdated();
```

Comment:
Can you look into whether this should be a const function returning a const ref?

Reply:
Getting something in order to set it may as well just be a public field.

Reply:
I don't think you can make this a function that takes a callable and still use an unreal delegate.